services/jobserver.py

```diff
@@ -13,7 +13,7 @@
 JOB_SERVER_TOKEN = env.str("JOB_SERVER_TOKEN")
 RELEASES_URL = urljoin(
     JOB_SERVER_URL,
-    "api/v2/releases/workspace/test-interactive",
+    "api/v2/releases/workspace/opensafely-interactive",
```

Comment:
Can I suggest we parameterise this as JOB_SERVER_WORKSPACE?

We need it to contruct multiple API urls (releases, submit, possibly others)?

e.g.
```
JOB_SERVER_WORKSPACE = env.str("JOB_SERVER_WORKSPACE")
RELEASES_URL = f"{JOB_SERVER_URL}/api/v2/releases/workspace/{JOB_SERVER_WORKSPACE}"
```

Reply:
Yeah, the workspace is repeated in JOBS_JOB_SERVER_JOBS_URL

Having these as separate configurations means we could get the out of sync, and submit jobs to one workspace, but look for releases in the other.

I think we do need to change bot JOB_SERVER_JOBS_URL and RELEASES_URL to be derived from JOB_SERVER_API and JOB_SERVER_WORKSPACE.

Reply:
That makes sense. It will make testing easier in future too.

Reply:
Right, just pushed some updates. The most recent commit shows the update to JOB_SERVER_JOBS_URL. It feels a bit messy, especially since the project is still in the URL. Not sure if it's just easier to keep the full hard-coded url in dotenv-sample where it's easier to see? Either way I guess it's likely to change soon.

Reply:
Yes, once we link to the job-server API to submit jobs, rather than for a slack link, then it won't have the project in the url anymore.

FWIW, I've discussed with George before removing the project from the url, as workspaces are globally unique anyway